libproject/devstart.rb

```diff
@@ -0,0 +1,90 @@
+require "optparse"
+require "fileutils"
+require_relative "../aou-utils/utils/common"
+require_relative "../aou-utils/serviceaccounts"
+require_relative "../aou-utils/workbench"
+require_relative "../aou-utils/swagger"
+
+# Update this whenever we want to generate libraries for the latest version of the workbench API.
+SWAGGER_SPEC = 'https://raw.githubusercontent.com/all-of-us/workbench/fac5e45d578d1e5f6d735824a89b1d4f5a8f8f37/api/src/main/resources/workbench.yaml'
```

Comment:
So I just learned that you can use tags in addition to sha1s for this purpose:

`https://raw.githubusercontent.com/all-of-us/workbench/stable-v1-rc1/api/src/main/resources/workbench.yaml`

This would allow you to further deconstruct this to be more readable:
```
YAML_RELEASE = 'stable-v1-rc1'
SWAGGER_SPEC = '...'.format(YAML_RELEASE)
```

Reply:
OK, I created a new tag on workbench (api_v1_1) and used it here.